python/lsst/meas/base/applyApCorr.py

```diff
@@ -148,12 +180,13 @@ def run(self, catalog, apCorrMap):
             self.log.info("Use complex flux sigma computation that double-counts photon noise "
                           "and thus over-estimates flux uncertainty")
```

Comment:
While we're here...could you change the above two log lines to `debug` level?  This seems a bit chatty.